client/packages/requisitions/src/ResponseRequisition/ListView/CreateRequisitionModal.tsx

```diff
@@ -30,28 +32,35 @@ interface CreateRequisitionModalProps {
     newRequisition: NewGeneralRequisition | NewProgramRequisition
   ) => void;
 }
-export const CreateRequisitionModal: FC<CreateRequisitionModalProps> = ({
-  isOpen,
-  onClose,
-  onCreate,
-}) => {
+export const CreateRequisitionModalComponent: FC<
+  CreateRequisitionModalProps
+> = ({ isOpen, onClose, onCreate }) => {
   const t = useTranslation();
-  const { data: programSettings, isLoading } =
-    useResponse.utils.programSettings();
+
+  const { data, isLoading } = useName.document.customers();
```

Comment:
Select customers in a parent component which is an independent query to program_requisition_settings query. 

Now call this separately based on selected customer.